cargo-flash/src/main.rs

```diff
@@ -124,21 +124,30 @@ fn main_try() -> Result<(), failure::Error> {
         args.remove(index);
     }
 
-    Command::new("cargo")
+    let status = Command::new("cargo")
         .arg("build")
         .args(args)
         .stdout(Stdio::inherit())
         .stderr(Stdio::inherit())
         .spawn()?
         .wait()?;
 
+    if !status.success() {
+        std::process::exit(1);
```

Comment:
This could also propagate the exit code of Cargo.

Reply:
Good point!